rust-toolchain

```diff
@@ -0,0 +1 @@
+1.75
```

Comment:
I think a `rust-toolchain.toml` is more idiomatic now no?

Reply:
Updated.